WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -128,6 +128,38 @@ $venue-map-wrapper-height: 400px;
 
 .competitions-table {
 
+  p {
+    margin: 0;
+  }
+
+  tr {
+    &.past {
+      a:link,
+      a:visited {
+        color: #139;
+      }
+    }
+
+    &.not-past {
+      a:link,
+      a:visited {
+        color: #e64503;
+      }
+    }
+  }
+
+  .big-column {
+    width: 100%;
+  }
+
+  td,
+  th {
+    white-space: nowrap;
```

Comment:
Instea of `big-column` and this `td, th` rule, could you use `.table-greedy-last-column` instead?